docs/source/v2/custom-database-names.blade.md

```diff
@@ -7,7 +7,7 @@ section: content
 
 # Custom Database Names {#custom-database-names}
 
-To set the a database name for a tenant, use set `_tenancy_db_name` key in the tenant's storage.
+To set a specific database name for a tenant, specify the `_tenancy_db_name` key in the tenant's storage.
```

Comment:
specific/specify

How about "To set a specific database name for a tenant, set the `_tenancy_db_name` key ..."? Now `set` is repetitive, but it's shorter so IMO it sounds better.

Reply:
yup